source/git.js

```diff
@@ -353,7 +356,7 @@ git.commit = function(repoPath, amend, message, files) {
   if (message === undefined)
     return task.setResult({ error: 'Must specify commit message' });
 
-  if ((!(files instanceof Array) || files.length == 0) && !amend)
+  if ((!(Object.prototype.toString.call(files) === '[object Array]') || files.length == 0) && !amend)
```

Comment:
Just curious: how is this better? Isn't `instanceof` the correct way of checking types in JavaScript?